### DISCUSSION TITLE
Determining if an image needs rotating

### DISCUSSION THREAD

**Discussion by pgscannell** [ImageMagick/ImageMagick#2292] [Help]
I know that this command line 
magick.exe -format '%[exif:orientation]' file-path
will return a '1' if the image is properly rotated and anything else if it isn't.

I proved this by actually running the command line from a command prompt.  I then modified the command so it would redirect its output to a file.  That works from within my file system but not too well from within my vbScript file.

I believe that there is a com object available to me from within this file:  ImageMagickObject.dll but I don't know how to activate it from within my vbScript code.

Anyone able to assist me on this?

Thanks,
Paul

**Comment by urban-warrior** [comment]
You may want to consider Magick.Net for your solution as it is well supported and well documented.  We provide ImageMagickObject as a convenience but the user base for ImageMagickObject is sparse.
